ui/src/app/subnets/views/FormActions/FormActions.tsx

```diff
@@ -0,0 +1,41 @@
+import AddFabric from "./components/AddFabric";
+import AddSpace from "./components/AddSpace";
+
+import { SubnetForms } from "app/subnets/enum";
+import type { SubnetForm } from "app/subnets/types";
+
+const FormComponents: Record<
+  SubnetForm,
+  ({ activeForm, setActiveForm }: FormActionProps) => JSX.Element | null
+> = {
+  [SubnetForms.Fabric]: AddFabric,
+  [SubnetForms.VLAN]: () => null,
+  [SubnetForms.Space]: AddSpace,
+  [SubnetForms.Subnet]: () => null,
+};
+
+export interface FormActionProps {
+  activeForm?: SubnetForm;
+  setActiveForm: React.Dispatch<React.SetStateAction<SubnetForm | undefined>>;
+}
+
+const FormActions = ({ activeForm, setActiveForm }: FormActionProps) => {
```

Comment:
Just missing return types on these function components.
``` ts
const FormActions = ({
  activeForm,
  setActiveForm,
}: FormActionProps): JSX.Element => {
```

Reply:
Cool, I generally prefer TS to infer the return type, but can add a return type explicitly.

Shall I add an eslint rule to enforce this (or at least warn about it)?
https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/explicit-function-return-type.md

Reply:
I thought it was enforced already? Hmm, I wonder if one of the recent dependency updates included an eslint plugin that removed the rule. 😕

Anyway yeah it should be enforced to be consistent with our tsconfig - at the moment the build is failing due to these missing return types.

Reply:
Right the rule was actually this one:
https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/explicit-module-boundary-types.md